bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Setup](#setup) \
+   2.1. [Setup Node and Configure](#setup-node-and-configure) \
+   2.2. [Build node from `x` tag](#build-node-from-x-tag) \
+   2.3. [Initialize the chain](#initialize-the-chain) \
+   2.4. [Update the configuration](#update-the-configuration) \
+   2.5. [Retrieve the genesis file](#retrieve-the-genesis-file) \
+   2.6. [Create a Keyring](#create-a-keyring) \
+   2.7. [Get Funds](#get-funds) \
+   2.8. [What is BLS Voting](#what-is-bls-voting) \
+   2.9. [Create BLS Key](#create-bls-key)
+3. [Run Node](#run-node)
+4. [Create Validator](#create-validator)
+5. [Verify your Validator](#verify-your-validator)
+
+## Prerequisites
+
+This system spec has been tested by validators and found to be comfortable:
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+You can run Babylon on lower-spec hardware for each component, but you may find that it is not highly performant or prone to crashing.
+## Setup
+
+We need to setup the following before we can classify as a validator. 
+- Setup node and configure
+- Create keyring
+- Get funds
+- Create BLS Key
+- Run node
+
+### Setup Node and Configure
+
+<!-- TODO: check add in the correct tag for the testnet -->
+#### Build node from `x` tag. 
+
+Here we will checkout the tag in which we will use. 
+
+` git checkout x`
+
+#### Initialize the chain
+
+Run  `babylond init --chain-id bbn-test-5` in the terminal 
+
+It should generate all of the necessary files such as `app.toml`, `client.toml`, `genesis.json`
+
+#### Update the configuration
+
+We need to then change the configuration details in the following files. 
+- In `app.toml` under `[btc-network]`change network to `signet` and `iavl-cache-size = 0`
+- In the `config.toml` add in the `seeds` and `persistent-peers` token (this can be the same token)
+
+#### Retrieve the genesis file 
+
+We next want to add in the genesis file from thee network. How we can do this is either copy the genesis file from  `https://rpc.devnet.babylonlabs.io/genesis` or we can use the terminal.
+
+```
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2  
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2  
+mv genesis.json ~/.babylond/config/genesis.json
+```
+
+This file needs to overwrite the existing genesis file in the `~/.babylond/config/genesis.json` and please ensure that the `chain-id` is correct.
+
+#### Create a Keyring
+
+What we want to do next is create a key we can do this by running:
+
+`babylond --keyring-backend test keys add <name>`
+
+We use `--keyring-backend test`, which specifies which backend to use for the keyring, `test` stores keys unencrypted on disk. At the end we add a `name` which is the name in which we will give to the key. 
```

Comment:
Does the validator have to use a `test` keyring? If not, what other options do they have? 
```suggestion
We use `--keyring-backend test`, which specifies which backend to use for the keyring, `test` stores keys unencrypted on disk. The `<name>` specifies a unique identifier for the key.
```

Reply:
Is this something we should denote?